sky/global_user_state.py

```diff
@@ -516,7 +545,9 @@ def get_cluster_from_name(
         # we can add new fields to the database in the future without
         # breaking the previous code.
         (name, launched_at, handle, last_use, status, autostop, metadata,
-         to_down, owner, cluster_hash) = row[:10]
+         storage_mounts_metadata, to_down, owner, cluster_hash) = row[:11]
```

Comment:
The new column will only be able to be added to the end of the columns. Can we make sure this is correct when the `clusters` table existed before this PR will work?

Reply:
confirmed that it works